src/sdk/pynni/nni/nas/pytorch/search_space_zoo/darts_search_space.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from collections import OrderedDict
+
+import torch
+import torch.nn as nn
+
+import ops
+from nni.nas.pytorch import mutables
+
+
+class DartsSearchSpace(nn.Module):
```

Comment:
I think DARTS search space is a cell rather than a full model. Suggest to changing into `DartsStackedCells` that accepts a cell class to build a full model.

Reply:
on the way

Reply:
> I think DARTS search space is a cell rather than a full model. Suggest to changing into `DartsStackedCells` that accepts a cell class to build a full model.

please review new commit https://github.com/microsoft/nni/pull/2589/commits/99c841bc8ebb8622e14709bb2d6d72f23af5c415